src/addQuestion/index.js

```diff
@@ -0,0 +1,94 @@
+import { useState } from "react";
+import { connect } from "react-redux";
+import { createQuestion } from "../store/question.js";
+import { questionFactory } from "../question.js";
+
+import "./index.css";
+
+let AddQuestionForm = ({ onSave, positionClass }) => {
+  const [question, setQuestion] = useState("");
+  const [askee, setAskee] = useState("");
+  const [status, setStatus] = useState("Unanswered");
+
+  const onChangeQuestion = ({ target: { value = "" } = {} } = {}) =>

```

Comment:
I usually make a single abstract function called set that abstracts the target extraction. The set function takes the setter function (it's a higher order function).